front/src/components/HeaderMenu/HeaderMenuContainer.tsx

```diff
@@ -92,78 +92,78 @@ function HeaderMenuContainer() {
       },
       {
         text: "Beneficiaries",
-        requiredAbp: ["create_beneficiaries"],
+        requiredAbps: ["create_beneficiaries"],
         links: [
           {
             link: `${oldAppUrlWithBase}&action=people_add`,
             name: "Add Beneficiary",
-            requiredAbp: ["create_beneficiaries"],
+            requiredAbps: ["create_beneficiaries"],
             external: true,
           },
           {
             link: `${oldAppUrlWithBase}&action=people`,
             name: "Manage Beneficiaries",
-            requiredAbp: ["manage_beneficiaries"],
+            requiredAbps: ["manage_beneficiaries"],
             external: true,
           },
         ],
       },
       {
         text: "Free Shop",
-        requiredAbp: ["checkout_beneficiaries"],
+        requiredAbps: ["checkout_beneficiaries"],
```

Comment:
Can you turn this into 
`[["view_inventory", "checkout_beneficiaries", "manage_tokens"]]`
?